lib/network/modules/components/nodes/util/CircleImageBase.js

```diff
@@ -14,12 +14,22 @@ class CircleImageBase extends NodeBase {
   }
 
   setImages(imageObj, imageObjAlt) {
-    if (imageObj) {
-      this.imageObj = imageObj;
+    if (!this.selected) {
+      if (imageObj) {
+        this.imageObj = imageObj;
 
-      if (imageObjAlt) {
-        this.imageObjAlt = imageObjAlt;
+        if (imageObjAlt) {
```

Comment:
If I remove those checks and create an image node only containing an `unselected` image or even no image (which is stupid but possible to do) then we get a JS error.
You may want to update the code yourself.